docs/references/phpdoc/tags/param.rst

```diff
@@ -1,57 +1,53 @@
 @param
 ======
 
-The @param tag is used to document a single argument of a function or method.
+The ``@param`` tag is used to document a single `argument of a function or method`_.
 
 Syntax
 ------
 
+.. code-block::
+
     @param [Type] [name] [<description>]
 
 Description
 -----------
 
-With the @param tag it is possible to document the type and function of a
-single argument of a function or method. When provided it MUST contain a
-Type to indicate what is expected; the description on the other hand is
-OPTIONAL yet RECOMMENDED in case of complicated structures, such as associative
-arrays.
+With the ``@param`` tag it is possible to document the :doc:`Type <../types>`
+and function of a single `argument of a function or method`_. When provided it
```

Comment:
`function` already is a domain term, so maybe `intent` of the arg would be better wording?